worldedit-core/src/main/java/com/fastasyncworldedit/core/command/tool/brush/ScatterBrush.java

```diff
@@ -64,8 +65,9 @@ public void build(EditSession editSession, BlockVector3 position, Pattern patter
             length = 1;
             visited.add(position);
         }
-        LocalBlockVectorSet placed = new LocalBlockVectorSet();
-        placed.setOffset(position.getX(), position.getZ());
+        BlockVector3 patternSize = pattern.size();
+        BlockVector3Set placed = BlockVector3Set.getAppropriateVectorSet(patternSize.add(distance, distance, distance));
+        placed.setOffset(position.getX(), (position.getY() + patternSize.getY()) / 2, position.getZ());
```

Comment:
That's probably meant to be 
```suggestion
        placed.setOffset(position.getX(), position.getY() + patternSize.getY() / 2, position.getZ());
```
but probably the pattern size isn't really beneficial here at all as it's not clear if it's all above or all below or something in between?

Reply:
Yes that's true